dist/actions/backup-environment/index.js

```diff
@@ -1726,6 +1727,14 @@ function whoAmI(parameters, runnerParameters) {
             logger.log("The Authentication Result: " + authenticateResult);
             const pacResult = yield pac("org", "who");
             logger.log("WhoAmI Action Result: " + pacResult);
+            const envIdLabel = "Environment ID:";
+            // HACK TODO: Need structured output from pac CLI to make parsing out of the resulting env id more robust
```

Comment:
@devkeydet, I was talking about the files with these lines of code. Under files changed I saw 26 modififcations with the same code across many files but with a TODO comment about needing better output from PAC CLI to make parsing easier. The snippet appears in 16 locations from a quick regex search on GitHub.

Reply:
Sorry for the confusion I was on my mobile device earlier and didn't see the code tag feature there. A bit hard to follow text without some link to the code snippet :'(